test-helpers.js

```diff
@@ -0,0 +1,25 @@
+function equal(actual, expected, message) {
+  if (actual === expected) {
+    const defaultMessage = `Expected ${expected} and received ${actual}`;
+    console.info("Pass: " + (message || defaultMessage));
+  } else {
+    const defaultMessage = `Expected ${expected} but received ${actual} instead`;
+    console.error("Fail: " + (message || defaultMessage));
+  }
+}
+
+function notEqual(actual, expected, message) {
+  if (actual !== expected) {
+    const defaultMessage = `${expected} is different to ${actual}`;
+    console.info("Pass: " + (message || defaultMessage));
+  } else {
+    const defaultMessage = `${expected} is the same as ${actual}`;
+    console.error("Fail: " + (message || defaultMessage));
+  }
+}
+
+function test(name, testFunction) {
```

Comment:
This has given us an awesome start for when we are ready on testing YAAAAAY 🏎️ 👾 ✨